django_airavata/wagtailapps/base/migrations/0020_auto_20190412_1655.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hi Keerthi, you can leave this one out of the PR. I'll investigate why this migration is getting generated.